documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -18,7 +18,18 @@ The `file-store` element will now install a SoftIndexFileStore instead of the pr
 implementation. Any configuration with a schema of 12 or older will automatically migrate the old
 data storage to the new automatically.
 
-= Upgrading from 12.0 to 12.1
+= Upgrading from 11.x or 12.0 to 12.1
+
+  If you are upgrading from 11.x to 12.1, you should also review the 12.0 to 12.1 upgrade notes.
+
+== Upgrading Deployments with Single File Cache Store Persistence
+
+When upgrading Infinispan from 11.x to 12.1, caches that include a `SingleFileStore` persistence configuration can encounter an issue that leads to data corruption.
+This issue affects Infinispan 12.1.0.Final to 12.1.4.Final.
```

Comment:
I believe it affects 12.0.x

Reply:
:+1: . I was thinking that as `12.0.x` wasn't supported we didn't have to worry about it, but a user doing `11.x` -> `12.0.x` -> `12.1.6.Final` could run into this issue as well.